deploy/cdk/cdk.context.json

```diff
@@ -94,7 +94,7 @@
   "viewer:hostnamePrefix": "viewer-iiif",
   "viewer:appRepoOwner": "ndlib",
   "viewer:appRepoName": "marble-image-viewer",
-  "viewer:appSourceBranch": "master",
+  "viewer:appSourceBranch": "update-new-pipeline",
```

Comment:
Did you intend to have this change in this PR?

Reply:
changed